src/components/PeoplePicker/PeoplePicker.tsx

```diff
@@ -176,7 +176,8 @@ export class PeoplePicker extends React.Component<IPeoplePickerProps, IPeoplePic
   private _onBlurCapture(ev) {
     // onBlur, relatedTarget refers to the element that got focus
     if (!this.refs.searchField.contains(ev.relatedTarget)
-      && !this.refs.pickerResults.contains(ev.relatedTarget) ) {
+      && !this.refs.pickerResults.contains(ev.relatedTarget)
+      && ev.currentTarget !== window ) {
       this._dismissPeoplePicker();
```

Comment:
lets flip this and check for losing focus to something not in the picker in order to dismiss.